opensearchpy/_async/client/security.py

```diff
@@ -0,0 +1,575 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+# ----------------------------------------------------
```

Comment:
@dblock, please also suggest if any changes are required here. This is autogenerated.

Reply:
I will raise one final PR for python generator with changes.